src/Histogram.ts

```diff
@@ -17,10 +17,15 @@ type HistogramData = {
 export default class Histogram {
   // no more than 2^32 pixels of any one intensity in the data!?!?!
   private bins: Uint32Array;
+  /** Min value in the original raw data. */
   private min: number;
+  /** Max value in the original raw data. */
   private max: number;
+  /** Size of each histogram bin in the scale of the original raw data. */
   private binSize: number;
+  /** Index of the first nonzero bin with at least 1 sample */
```

Comment:
what does "nonzero bin with at least 1 sample" mean?  Are "nonzero" and "at least 1 sample" meant to mean the same thing?

Reply:
This comment was copied from the constructor (`// track the first and last nonzero bins with at least 1 sample`). It used to represent the first bin (excluding the zero-th bin, back when values of `0` were placed into the zero-th bin) that had at least one pixel of that intensity.

Now that I think about it, I think this code is obsolete? `dataMinBin` and `dataMaxBin` can be safely removed, and `getMin()` and `getMax()` can just return 0 and `NBINS - 1` respectively.

Reply:
(The comment I had on `getMin()` and `getMax()` were actually incorrect for the current behavior, but returning `0` and `NBINS - 1` *should* be the correct behavior.)